packages/api/src/entity/library_item.ts

```diff
@@ -196,4 +196,28 @@ export class LibraryItem {
 
   @Column('text', { nullable: true })
   recommenderNames?: string[] | null
+
+  @Column('timestamptz')
+  hiddenAt?: Date | null
+
+  @Column('timestamptz')
+  sharedAt?: Date | null
+
+  @Column('text')
+  sharedBy?: string | null
```

Comment:
We could maybe leave out sharedAt and sharedBy right now.

Reply:
Why do we want to leave out them?

I need `sharedAt` to know if this is in following and `sharedBy` is to store the feed url or followee's username.

Reply:
@sywhb oh i see. I was thinking it was for posting recommendations, not data needed by the feeds.

Reply:
I didn't come up with good variable names here

Reply:
@jacksonh maybe I should change `sharedAt` to `addedToFollowingAt` and `sharedBy` to `addedToFollowingBy`?

Reply:
Yeah I think that would make it a little easier to understand